### PR TITLE
Update piu+scp docs to account for instance connect changes

### PR DESCRIPTION
**Pull request by linki**
When using instance connect the old docs for scp didn't work anymore. Specifying both usernames fixed the problem.

### CODE REVIEW

File: user-guide/ssh-access.rst [hunk 1]
```diff
@@ -67,6 +67,12 @@ copying files from and to the EC2 instance appears unnecessary hard at first.
 
 Luckily OpenSSH's ``scp`` supports jump hosts with the ``ProxyCommand`` configuration option:
 
+.. code-block:: bash
+
+    $ scp -o ProxyCommand="ssh -W %h:%p odd@odd-eu-west-1.myteam.example.org" mylocalfile.txt ubuntu@172.31.146.1:
```
Comment (mikkeloscar) [inline user-guide/ssh-access.rst]: Note you can also use the `-J` flag like this:

```sh
scp -J odd@odd-eu-west-1.myteam.example.org mylocalfile.txt ubuntu@172.31.146.1:
```

Comment (linki) [reply user-guide/ssh-access.rst]: Which version do you have? `-J` isn't available for my `scp`. However, I could reduce it a little bit based on https://www.madboa.com/blog/2017/11/02/ssh-proxyjump/

Comment (mikkeloscar) [reply user-guide/ssh-access.rst]: I see, I have this version:

```
ssh -V
OpenSSH_8.3p1, OpenSSL 1.1.1h  22 Sep 2020
```

But better to use the version that works with older versions ofc.

Comment (linki) [reply user-guide/ssh-access.rst]: @mikkeloscar Thanks for checking. For reference mine is:

```
OpenSSH_7.6p1 Ubuntu-4ubuntu0.3, OpenSSL 1.0.2n  7 Dec 2017
```

Let's keep it at the more compatible syntax for now.

**Pull request closed without merging**